tools/common/common-build.targets

```diff
@@ -21,12 +21,13 @@
       <_ImportLibraryFileNames Include="@(Link -> '%(ImportLibrary)')">
       </_ImportLibraryFileNames>
       
-      <_BuiltProjectOutputGroupOutput Include="@(BuiltProjectOutputGroupOutput -> '%(FinalOutputPath)')" />
-      <PackageFile Include="@(_BuiltProjectOutputGroupOutput)" Condition="Exists('%(FullPath)')">
+      <!-- EXPLORATORY DIFF to reduce input duplication suspected to be the cause of duplicate file errors -->
+      <_IWBuiltProjectOutputGroupOutput Include="@(BuiltProjectOutputGroupOutput -> '%(FinalOutputPath)')" />
+      <PackageFile Include="@(_IWBuiltProjectOutputGroupOutput->Distinct())" Condition="Exists('%(FullPath)')">
         <PackagePath>$(PackageContentBasePath)\%(Filename)%(Extension)</PackagePath>
       </PackageFile>
       
-      <PackageFile Include="@(_ImportLibraryFileNames->Distinct())" Condition="Exists('%(FullPath)')">
+      <PackageFile Include="@(_ImportLibraryFileNames -> '%(FullPath)')" Condition="Exists('%(FullPath)')">
```

Comment:
><PackageFile Include="@(_ImportLibraryFileNames -> '%(FullPath)')" Condition="Exists('%(FullPath)')"> [](start = 6, length = 101)

This almost assuredly wrong. The _ImportLibraryFileNames itemgroup is built up out of every item in the link groups ImportLibrary property. Because of project dependencies and other things getting linked in, there are almost assuredly more than one item in Link so I added Distinct() to ensure we only get one. If you feel you need the FullPath property transformation, i'd add another intermediate item group